test/integration/integration_test.go

```diff
@@ -264,6 +270,16 @@ func FeatureContext(s *godog.Suite) {
 			runner.RunCommand("addons list")
 		}
 
+		var splittedCommands []string
+		if execBeforeFeature != "" {
+			splittedCommands = strings.Split(execBeforeFeature, delimiterConst)
```

Comment:
👍

Reply:
@praveenkumar Thanks for the suggestion BTW, like it, I need to put it into my mind as common practice.

Reply:
👍